geode-core/src/test/java/org/apache/geode/test/dunit/rules/ClusterStartupRule.java

```diff
@@ -284,39 +302,22 @@ public MemberVM startServerAsEmbededLocator(int index, Properties properties) th
     return memberVM;
   }
 
-  public void stopVM(int index) {
-    stopVM(index, true);
-  }
-
-  public void stopVM(int index, boolean cleanWorkingDir) {
-    VMProvider member = occupiedVMs.get(index);
-
-    if (member == null)
-      return;
-
-    member.stopVM(cleanWorkingDir);
+  public MemberVM startServerAsEmbeddedLocator(int index, Properties properties)
+      throws IOException {
+    return startServerAsEmbeddedLocator(index, properties, VersionManager.CURRENT_VERSION);
   }
 
-  public ClientVM startClientVM(int index, String username, String password,
```

Comment:
Whoops.  I had refactored this out during my initial exploration, but had meant to bring it back before opening the PR.

Reply:
yeah, this method would be helpful to stay here.